qa/tasks/systemd.py

```diff
@@ -69,7 +69,7 @@ def task(ctx, config):
         mon_role_name = 'mon.' + name
         mds_role_name = 'mds.' + name
         mgr_role_name = 'mgr.' + name
-        m_osd = re.search('--id (\d+) --setuser ceph', r.stdout.getvalue())
+        m_osd = re.search('--id (\d+) --setuser ceph', ps_eaf)
```

Comment:
this is a bug fix. need to backport it to nautilus and octopus.

Reply:
I guess whole PR should be backported since we need these tests both in nautilus and octopus

Reply:
why?

Reply:
you ask why do we need static analysis and unit tests for qa/tasks in nautilus and octopus?

Reply:
yes, why?

Reply:
because we want to have PR tested?

Reply:
even backports, or my understanding is wrong?

Reply:
i have no interests in backporting the tests for tests.

Reply:
Can I do that?

Reply:
the reason we would want this, because it can help to find bugs in tests early, not waiting until teuthology reports a failure, we safe engineering time and computing resources in the lab, I would also move this tox check out of make check or make it as separate jenkins job to find qa/tasks issue even faster.

Reply:
sure. please feel free to do so. i just don't think it is worth the efforts